exercises/practice/state-of-tic-tac-toe/.docs/instructions.md

```diff
@@ -0,0 +1,101 @@
+# Instructions
+
+In this exercise, you're going to implement a program that determines the state of a [tic-tac-toe][] game.
+(_You may also know the game as "noughts and crosses" or "Xs and Os"._)
+
+The games is played on a 3×3 grid.
```

Comment:
```suggestion
The game is played on a 3×3 grid.
```

This suggested change is to mirror a recent update in the problem specifications to fix this grammar (exercism/problem-specifications#2482).

Reply:
Done